samples/net/gptp/boards/nucleo_f767zi.conf

```diff
@@ -0,0 +1,3 @@
+# STM32 driver settings
+CONFIG_ETH_STM32_HAL=y
```

Comment:
This should not be needed, driver will be enabled based on `CONFIG_NETWORKING=y` and `&mac {status = "okay";};` combination.
Apply to all

Reply:
Good catch - just pushed that in a separate commit, please confirm that this is what you had in mind.

Reply:
That's fine, but please fix directly in the initial guilty commit.